NOTICE

```diff
@@ -1 +1 @@
-Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Is ... this up to date?

Reply:
clearly not :) I really should just get tid of these NOTICE files entirely. They are pointless. This was just copy/pasta from the original template repository...